renderer/blockquote.go

```diff
@@ -150,7 +165,27 @@ func ParseBlockQuoteType(node ast.Node, source []byte) BlockQuoteType {
 		return ast.WalkContinue, nil
 	})
 
-	return t
+	return t, ct, context
+}
+
+// Helper function to update the source in place
+func ghaAlertsUpdateSourceInPlace(source []byte, node ast.Node, value string) {
+	leftNode := node.PreviousSibling().(*ast.Text)
+	midNode := node.(*ast.Text)
+	rightNode := node.NextSibling().(*ast.Text)
+
+	title := string(midNode.Value(source))
+	sanitizedTitle := strings.Replace(strings.ToLower(title), "!", "", -1)
```

Comment:
Instead of doing this on a character base, can we do it on a word base (i.e. look for the existing ghalerts with a [! in front and ] afterwards and then simply replace what you found?